src/_gettsim/demographic_vars.py

```diff
@@ -330,3 +332,30 @@ def birthdate_decimal(
     out = geburtsjahr + (geburtsmonat - 1) / 12
 
     return out
+
+
+@policy_info(skip_vectorization=True)
+def anz_paare_hh(
```

Comment:
This is a quick and dirty solution for the problem. We may want to generalize this, but I couldn't come up with other targets where this logic could be relevant.